service/frontend/workflow_handler.go

```diff
@@ -4145,9 +4147,10 @@ func (wh *WorkflowHandler) unaliasStartWorkflowExecutionRequestSearchAttributes(
 	}
 
 	// Shallow copy request and replace SearchAttributes fields only.
-	newRequest := *request
+	// FIXME actually shallow copy
```

Comment:
I left these in here in case we needed shallow copies for performance. Once I've gone through the performance testing I've kicked off I'll see if these are necessary (they probably aren't) and address them appropriately at that point.

I'm going to change these to `// NOTE` instead of fixmes

Reply:
I actually just deleted my note entirely. I don't think it's necessary, and were this actually a problem we'd find it easily in a profile